docker-compose.yml

```diff
@@ -465,4 +465,4 @@ services:
       - MYSQL_DATABASE=MyAppDb
       - SPRING_PROFILES_ACTIVE=docker
     ports:
-      - "3306:3306"
+      - "3309:3306"
```

Comment:
I'm curious on why this was changed

Reply:
I had changed it because git bash when I was doing docker compose up it would tell me I had a problem with my ports and somebody else in the class had that same issue and they told me to change the port, and it worked, but I forgot to change it back to 3306 when I was doing the push. Thanks for letting me know